infra/cifuzz/fuzz_target.py

```diff
@@ -92,7 +92,30 @@ def fuzz(self):
     if not test_case:
       logging.error('No test case found in stack trace.', file=sys.stderr)
       return None, None
-    return test_case, err_str
+    if self.is_reproducible(test_case):
+      return test_case, err_str
+    logging.error('A crash was found but it was not reproducible.')
+    return None, None
+
+  def is_reproducible(self, test_case):
+    """Checks if the test case reproduces.
+
+      Args:
+        test_case: The path to the test case to be tested.
+
+      Returns:
+        True if crash is reproducible.
+    """
+    command = [
+        'docker', 'run', '--rm', '--privileged', '-v',
+        '%s:/out' % os.path.dirname(self.target_path), '-v',
+        '%s:/testcase' % test_case, '-t', 'gcr.io/oss-fuzz-base/base-runner',
+        'reproduce', self.target_name, '-runs=100'
+    ]
+    _, _, err_code = utils.execute(command)
```

Comment:
We should do a round of retries, as there may be bugs that are reproduce e.g. 1/5 times and we still want to report them.

Reply:
Doesn't -runs=100 set it to retry 100 times? 

From the libfuzzer documentation:
```Random seed. If 0 (the default), the seed is generated.
-runs
Number of individual test runs, -1 (the default) to run indefinitely.```

Reply:
Set it to 10 rounds of checking as Max said that is how may clusterfuzz does.